### ISSUE TITLE
step03 can't deal with gif files

### ISSUE THREAD

**Issue by SteveGoldstein** [AFIDSI/PPL_OutlierPrediction#10]
It looks like step03 is chocking on producing the gif file.  Same error msg on olvi and  CHTC node..

```
ERROR: LoadError: Invalid file extension: /afs/cs.wisc.edu/u/s/t/stevegold/git/PPL_OutlierPrediction/src/step03-linear-spline/step03.gif
Stacktrace:
 [1] error(::String, ::String)
   @ Base ./error.jl:42
 [2] savefig(plt::Plots.Plot{Plots.GRBackend}, fn::String)
   @ Plots /p/DSI_PPL/juliaPkgs/packages/Plots/PGQ1Y/src/output.jl:123
 [3] savefig(fn::String)
   @ Plots /p/DSI_PPL/juliaPkgs/packages/Plots/PGQ1Y/src/output.jl:126
 [4] show(plot::Animation, filename::String)
   @ Main /afs/cs.wisc.edu/u/s/t/stevegold/git/PPL_OutlierPrediction/src/step01-importing-data/utilities/display.jl:34
 [5] top-level scope
   @ /afs/cs.wisc.edu/u/s/t/stevegold/git/PPL_OutlierPrediction/src/step03-linear-spline/step03.jl:179
in expression starting at /afs/cs.wisc.edu/u/s/t/stevegold/git/PPL_OutlierPrediction/src/step03-linear-spline/step03.jl:179
```

**Comment by SteveGoldstein** [comment]
step05-smooth.jl fails with the same error on olvi.  It hasn't been tested on CHTC.

**Comment by SteveGoldstein** [comment]
same error on step02.jl  -- and note the `show(..."step03.gif")` has the wrong filename.

**Comment by SteveGoldstein** [comment]
Same for step04.jl -- and note the `show(..."step05.gif")` function call has the wrong filename.

**Comment by SteveGoldstein** [comment]
Fixed by  870e1cd88ad
